### ISSUE TITLE
Wiki formatting suggestions

### ISSUE THREAD

**Issue by pietrygamat** [PixlOne/logiops#236]
The [Configuration](https://github.com/PixlOne/logiops/wiki/Configuration) page of the wiki is quite hard to read because of lack of indentation in markdown. I found it hard to follow and clearly distinguish between gestures and action, etc. 

I have modified the page slightly - added intendtaion and grouped all actions and gestures in their separate respective sections + added some Copy/Paste ready blocks so users have more examples to look at and get familiar with the use of quotation symbols and semicolons, that gave me so much hard time the first time I found this project.

Sadly, I do not know how to create PR to a wiki, so feel free to manually review the changes here: 
https://github.com/PixlOne/logiops/compare/master...pietrygamat:wiki.formatting?expand=1

**Comment by TrevCan** [comment]
Seems like a good idea, maybe an index / table of contents could be added at the top for better understanding. As per the wiki PR, I believe GitHub does not yet allow you to create a PR of the wiki unless you're committing to some branch in the main repo.
